app/locales/en.json

```diff
@@ -122,7 +122,16 @@
     "skip_this_step": "Skip this step",
     "team": "Team",
     "team_para": "Our team is composed of a consortium of epidemiologists, engineers, data scientists, digital privacy evangelists, professors and researchers from reputable institutions, including: MIT, Harvard, The Mayo Clinic, TripleBlind, EyeNetra, Ernst & Young and Link Ventures.",
-    "terms_of_use": "Terms of use"
+    "terms_of_use": "Terms of use",
+    "qr_exit": "Home",
```

Comment:
Could we pull all of these out into a `qr: {}` key? And then just have something like 
```
"qr": {
   exit: "Home", 
   ... 
}
```